docs/sites/site_navigation.md

```diff
@@ -36,26 +36,21 @@ Or you can add one of your [pages](../sites/site_pages), which should be listed
 
 ![URL block with the label “RRCHNM” and url chnm.gmu.edu](../sites/sitesfiles/sitenav_url.png) URL Block
 
-**Browse** blocks create a browse page for [resources](../sites/site_resources.md) already associated with your site. There are blocks to create a browse items page and a block to browse item sets.  
+**Browse** blocks create a browse page for [resources](../sites/site_resources.md) already associated with your site. There are blocks to create a browse items page and a block to browse item sets page.  
```

Comment:
There's quite a few references to "blocks" on this page. As far as I know, blocks are a feature of pages, not navigation. I'd use "links" instead of "blocks."

Reply:
Lots of those are legacy/existing verbiage, but I've corrected them all.